OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                two labels of two textile companies, one is glued into the inner
+                                part of the front board in <ref type="mss" corresp="ESqma005"/> and
+                                the other is sewn onto <locus target="#2v"/> in <ref type="mss"
+                                    corresp="BDLaethf29"/> (see <ref target="#e3"/>). <ref
+                                    type="mss" corresp="ESqma005"/> shows the label of “W&amp;I
+                                Peel”, the (later) company of the successful textile industrialist
+                                Robert Peel (1723–1795), widely known as <persName
+                                    ref="wd:Q16859468"/> (the label displays the Peel arms including
+                                a shuttle held by a lion, below a bee and the family motto
+                                'Industria'; it is the same as provided in Wikipedia (<ref
+                                    target="https://en.wikipedia.org/wiki/Parsley_Peel">, accessed
+                                    on 25 May 2020</ref>). Both labels are related most probably to
+                                the silk or cotton inlays in both manuscripts <ref type="mss"
+                                    corresp="BDLaethf29"/> and <ref type="mss" corresp="ESqma005"
+                                />.<lb/> We therefore may conclude that <ref type="mss"
+                                    corresp="BDLaethf29"/> and <ref type="mss" corresp="ESqma005"/>
+                                were manufactured around the same time, somewhen between 1841–1861,
+                                in the aera of <placeName ref="LOC3293GantaA"/>, <placeName
+                                    ref="LOC5932Tegray">Tǝgrāy</placeName> in Ethiopia by a
+                                professional scribe with connections to foreign traders or at least
+                                to imported goods. </p>
+                            <p>Coming back to <ref type="mss" corresp="BDLaethf29"/>: The name of
+                                the original patron or donor (introduced as <foreign xml:lang="gez"
+                                    >ገበርከ</foreign> ‘your servant’) is erased from the supplication
+                                formulas throughout the manuscript. Only the first part of his name
+                                    <foreign xml:lang="gez">ገብረ</foreign> survived. His name was
+                                replaced by that of <persName ref="PRS13017GabraMar"/>. The name of
+                                    <persName ref="PRS13021WaldaMik"/> written by the main hand is
+                                mentioned only once, on <locus target="#80r"/>.</p>
+                            <p>The manuscript must have come into the possession of the church of
+                                Maqdala Madḫāne ʾĀlam <ref target="INS0101MadhaneAlam"/> at a
```

Comment:
"Maqdala Madḫāne ʾĀlam `<ref target="INS0101MadhaneAlam"/>`" is a double reference. I would always be in favour of only having the marked up reference `<placeName ref="INS0101MadhaneAlam"/>`, but if you already know that you want to refer to it with "Maqdala Madḫāne ʾĀlam" and this is not in the institution record, it could be written as content of placeName.

Reply:
We will make the rules for what will be automatically retrieved in the catalogue. Of course, this will be easier when the records are consistently encoded :)